### PR TITLE
fix: use image prop from ai-studio.yaml when defined

### LINKED ISSUE #161: Add option to tag/name a resulting image when pulling a sample
user should have an option to tag/name the resulting image when pulling a sample. 
The images when built by ai studio are listed as docker.io/library/chatbot-example.

Should we add a field in the ai-yaml for image-name so it is used as default?

### PR DESCRIPTION
**Pull request by lstocchi**
### What does this PR do?

It adds support for the new image prop added to ai-studio.yaml file.

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

it resolves #161 

### How to test this PR?

1. run any recipe and check that the resulting image have the name defined in its ai-studio. You should delete your local recipe repo as it has to download the new one. If you uses an old repo nothing should change

### CODE REVIEW

File: packages/backend/src/managers/applicationManager.spec.ts [hunk 1]
```diff
@@ -1358,3 +1358,59 @@ describe('pod detection', async () => {
...
+      id: 'recipe1',
+    } as Recipe;
+    const container = {
+      name: 'name',
+      image: 'quay.io/repo/image',
+    } as ContainerConfig;
+    const imageTag = manager.getImageTag(recipe, container);
+    expect(imageTag).equals('quay.io/repo/image:latest');
+  });
+})
```
Comment (benoitf) [inline packages/backend/src/managers/applicationManager.spec.ts:1416]: looks the file is unformatted, don't know if we need pre-commit hooks like in podman-desktop

Comment (lstocchi) [reply packages/backend/src/managers/applicationManager.spec.ts]: yes i guess we should add it

Comment (lstocchi) [reply packages/backend/src/managers/applicationManager.spec.ts]: opened issue https://github.com/projectatomic/ai-studio/issues/603

**Comment by sallyom** [comment]
This will push the end-to-end story forward, making it possible to take the pod.yaml that ai-lab creates and `podman kube play pod.yaml` on any system. 

(as a follow-up, an option to push the ai-lab built images would be really convenient)

**Comment by jeffmaury** [comment]
Update to the packaging guide is missing

**Comment by lstocchi** [comment]
> Update to the packaging guide is missing

WDYM? I added the image prop description and updated the example -> https://github.com/projectatomic/ai-studio/blob/bdeceea78f90bd8769c71493368320f26b64fce8/PACKAGING-GUIDE.md

**Comment by jeffmaury** [comment]
> > Update to the packaging guide is missing
> 
> WDYM? I added the image prop description and updated the example -> https://github.com/projectatomic/ai-studio/blob/bdeceea78f90bd8769c71493368320f26b64fce8/PACKAGING-GUIDE.md

My fault, sorry

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by lstocchi** [comment]
> (as a follow-up, an option to push the ai-lab built images would be really convenient)

@sallyom Could you please open an issue so we don't forget about it?  Thank you!!

**Pull request merged**